idb/postgres/postgres.go

```diff
@@ -1659,8 +1669,10 @@ func (db *IndexerDb) GetAccounts(ctx context.Context, opts idb.AccountQueryOptio
 	if err != nil {
 		err = fmt.Errorf("account round err %v", err)
 		out <- idb.AccountRow{Error: err}
+		if rerr := tx.Rollback(ctx); rerr != nil {
```

Comment:
Could you move it below `close(out)` or above `err = ...` so as to not break the flow? There are some similar cases below.